htdocs/core/class/commonobject.class.php

```diff
@@ -7743,20 +7743,20 @@ public function showOptionals($extrafields, $mode = 'view', $params = null, $key
 					// Test on 'enabled' ('enabled' is different than 'list' = 'visibility')
 					$enabled = 1;
 					if ($enabled && isset($extrafields->attributes[$this->table_element]['enabled'][$key])) {
-						$enabled = dol_eval($extrafields->attributes[$this->table_element]['enabled'][$key], 1, 1, '1');
+						$enabled = dol_eval($extrafields->attributes[$this->table_element]['enabled'][$key], 1, 1, '0');
```

Comment:
Param '0' should be used only for 
$extrafields->attributes[$this->table_element]['computed']
not for
$extrafields->attributes[$this->table_element]['enabled']